app/components/app-header.hbs

```diff
@@ -3,13 +3,14 @@
   @homeRoute="home"
   @appTitle="Bobcat"
 >
-  <li class="au-c-list-horizontal__item">
+  {{!-- TODO: provide a valid help link --}}
```

Comment:
Will there even be a help link? 😄 Seems like a straightforward app. Unless it's a "rule" that every app needs one?

I would just delete the code. We can add it again if needed.